clowder-framework/clowder2
# Basic implementation of admin in user collection in mongodb

To test

1. you can use Postman or
2.  delete all users from mongodb and keycloak, then register. Please check mongodb, the user should be set as admin (admin=true)

To test that one user can set another as admin, please use POSTMAN. Refer to the attached ss for details.
<img width="532" alt="Screenshot 2023-10-25 at 1 36 18 PM" src="https://github.com/clowder-framework/clowder2/assets/22405928/03a2ffda-d272-48fe-832f-4bdb18a9678b">
